### DISCUSSION TITLE
Template repository

### DISCUSSION THREAD

**Discussion by davidedellagiustina** [typst/typst#407] [General]
I imagine that many people will soon start to develop and share their own templates for Typst. Is there already some common (and somewhat "official" place where people could upload/submit their templates for others to download? I'm thinking e.g. to something as simple as a GitHub repository with templates organized by category (e.g. article, book, presentation, resume, etc.), with people doing PRs to upload their stuff (this would also allow to track templates' author(s)).

**Comment by Enivex** [comment]
Here's a repository that might interest you 

https://github.com/qjcg/awesome-typst

**Reply by davidedellagiustina** [reply]
Very interesting, thanks!

**Reply by Zaitara** [reply]
It would be great if there was an online collection of templates like overleaf
